src/aws-healthomics-mcp-server/awslabs/aws_healthomics_mcp_server/utils/aws_utils.py

```diff
@@ -23,23 +23,28 @@
 from awslabs.aws_healthomics_mcp_server import __version__
 from awslabs.aws_healthomics_mcp_server.consts import DEFAULT_REGION
 from loguru import logger
-from typing import Any, Dict, Optional
+from typing import Any, Dict
 
 
-def get_aws_session(region: Optional[str] = None) -> boto3.Session:
-    """Get an AWS session with the specified or default region.
+def get_region() -> str:
+    """Get the AWS region from environment variable or default.
 
-    Args:
-        region: AWS region name (optional)
+    Returns:
+        str: AWS region name
+    """
+    return os.environ.get('AWS_REGION', DEFAULT_REGION)
+
+
+def get_aws_session() -> boto3.Session:
+    """Get an AWS session with the centralized region configuration.
 
     Returns:
         boto3.Session: Configured AWS session
     """
     botocore_session = botocore.session.Session()
     user_agent_extra = f'awslabs/mcp/aws-healthomics-mcp-server/{__version__}'
     botocore_session.user_agent_extra = user_agent_extra
-    region = region or os.environ.get('AWS_REGION', DEFAULT_REGION)
-    return boto3.Session(region_name=region, botocore_session=botocore_session)
+    return boto3.Session(region_name=get_region(), botocore_session=botocore_session)
```

Comment:
Maybe a better way to do this would be to get the region from the session and if that doesn't work then fall back to the default? That way we can make use of the SDKs standard region chain which accounts for all the ways you can configure a region.

```
import boto3

my_session = boto3.session.Session()
current_region = my_session.region_name
print(f"Current AWS Region: {current_region}")
```